src/main/java/nl/novi/Les10SpringBootTechItEasyApplication/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,46 @@
+package nl.novi.Les10SpringBootTechItEasyApplication.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class TelevisionsController {
+    @GetMapping("/televisions")
+    public ResponseEntity<Object> getTelevisions() {
+        return ResponseEntity.ok();
+    }
+
+    @GetMapping("/televisions/{id}")
+    public ResponseEntity<Object> getTelevisions(@PathVariable int id) {
+//        if(id>3) {
+//            throw new RecordNotFoundException("id > 3");
+//        }
+        return ResponseEntity.ok();
+    }
+
+    @PostMapping("/televisions")
+    public ResponseEntity<Object> postTelevision(@RequestBody String televisionName) {
+        return ResponseEntity.created();
+    }
+
+    @PutMapping("/television/{id}")
+    public ResponseEntity<Object> putTelevision(@PathVariable int id, @RequestBody String televisionName) {
+        return ResponseEntity.ok();
+    }
+
+    @DeleteMapping("/television/{id}")
+    public ResponseEntity<Object> deleteTelevision(@PathVariable int id) {
+        return ResponseEntity.noContent();
+    }
+}
+
+    /* een GET-request voor alle televisies
```

Comment:
Dit project is natuurlijk nog een "work in progress", maar probeer bij je eindopdracht geen comments mee te pushen.

Reply:
ik blijf voorloppig die wel meepushen, omdat deze comments zijn alleen maar voor mijn studie bedoeld zijn en onthouden waarom bepaalde dingen moeten worden gedaan of reeds gedaan.
Uiteraard als het om een echte werkcode zou zijn (bijvoorbeeld het eindopdracht), zal ik dan geen onnodige comments gebruiken